project-base/storefront/graphql/docs/schema.md

```diff
@@ -248,7 +248,8 @@ Returns article filtered using UUID or URL slug
 <td valign="top"><a href="#articleconnection">ArticleConnection</a>!</td>
 <td>
 
-Returns list of articles that can be paginated using `first`, `last`, `before` and `after` keywords and filtered by `placement`
+Returns list of articles that can be paginated using `first`, `last`, `before`
```

Comment:
Did you format this document by accident? Seems like an unwanted change...

Reply:
Nope, command `pnpm gql` was probably not run for some time and probably not after some upgrades. This is formatted just by library.

Reply:
I cannot reproduce it.